lib/aca_entities/families/family_member.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Families
+    class FamilyMember < Dry::Struct
+      # TODO: Person hbx_id until we add hbx_id on family member
+      attribute :hbx_id,                                      Types::String.optional.meta(omittable: false)
+      attribute :is_primary_applicant,                        Types::Bool.optional.meta(omittable: false)
+      attribute :is_consent_applicant,                        Types::Bool.optional.meta(omittable: false)
+      attribute :is_coverage_applicant,                       Types::Bool.optional.meta(omittable: false)
+      attribute :is_active,                                   Types::Bool.optional.meta(omittable: true)
+      attribute :applicant_reference,                         AcaEntities::MagiMedicaid::ApplicantReference.optional.meta(omittable: true)
```

Comment:
change attribute name to magi_medicaid_applicant_reference

Reply:
Updated to magi_medicaid_application_applicants to array